### PR TITLE
Feature/add query db security group public scope

### LINKED ISSUE #1536: Add db security group public scope query for Ansible
### Platform
Ansible
### Provider
AWS

### Description
The CIDR must be diferent than 0.0.0.0/0

### PR DESCRIPTION
**Pull request by cataraujo190**
Close #1536

### CODE REVIEW

File: assets/queries/ansible/aws/db_security_group_public_scope/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "aws_db_security_group_no_public_scope",
+  "queryName": "AWS DB Security Group with Public Scope",
+  "severity": "HIGH",
+  "category": "Network Security",
+  "descriptionText": "The CIDR must be diferent than 0.0.0.0/0",
+  "descriptionUrl": "https://docs.ansible.com/ansible/latest/collections/community/aws/rds_instance_module.html#parameter-db_security_groups"
+}
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_public_scope/metadata.json:8]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

File: assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        to_port: 80
+        cidr_ip: 10.1.1.1/32
+  rules_egress:
+      - proto: tcp
+        from_port: 80
+        to_port: 80
+        cidr_ip: 10.1.1.1/32
+        group_name: example-other
+        # description to use if example-other needs to be created
+        group_desc: other example EC2 group
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml:21]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

File: assets/queries/ansible/aws/db_security_group_public_scope/test/positive.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        # the containing group name may be specified here
+        group_name: example
+    rules_egress:
+      - proto: tcp
+        from_port: 80
+        to_port: 80
+        cidr_ip: 0.0.0.0/0
+        group_name: example-other
+        # description to use if example-other needs to be created
+        group_desc: other example EC2 group
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_public_scope/test/positive.yaml:57]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

File: assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+- name: example ec2 group2
+  ec2_group1:
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml:2]: I couldn't find a module named `ec2_group1`. Perhaps you meant `ec2_group`

File: assets/queries/ansible/aws/db_security_group_public_scope/metadata.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "aws_db_security_group_no_public_scope",
+  "queryName": "AWS DB Security Group with Public Scope",
+  "severity": "HIGH",
+  "category": "Network Security",
+  "descriptionText": "The CIDR must be diferent than 0.0.0.0/0",
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_public_scope/metadata.json:6]: `"AWS DB Security Group's CIDR must be different than 0.0.0.0/0"`

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
requesting changes

File: assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml (hunk 4 above)
Comment (cataraujo190) [reply assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml]: I changed it for testing. I was having an issue, with document results.

**Review by rogeriopeixotocx — APPROVED** [review]

**Pull request merged**